runtime/src/commonTest/proto/pbandk/testpb/custom_options.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package pbandk.testpb;
+
+import "google/protobuf/timestamp.proto";
+import "pbandk/testpb/validate.proto";
+
+message SingleRequiredCustomOption {
```

Comment:
Add a test for some of the other places where options can be added in addition to field options. For example, add a message option of `(testpb.disabled) = true` to a message (that one will test not only message options but also top-level primitive options).

Reply:
Hmm, I realized that your PR doesn't include support for outputting message options into the generated code (i.e. a field on `SingleRequiredCustomOption.Companion.descriptor.options` that references this `(testpb.disabled) = true` option that you've added.

Reply:
Oops, hit enter too soon... Did you want to add support for `MessageDescriptor.options` in this PR or leave it for a future PR?

Reply:
Dammit, not sure why I thought this was working but yeah very clearly I need to add plumbing. I'm going to say i'll do a follow on story as it's going to require multiple files changed and more testing. Great call out.